samples/drivers/eeprom/sample.yaml

```diff
@@ -2,20 +2,29 @@ sample:
   name: EEPROM Sample
 common:
     tags: eeprom
-    harness: console
     harness_config:
       type: one_line
       regex:
         - "Reset the MCU to see the increasing boot counter."
 
 tests:
   sample.drivers.eeprom:
-    tags: eeprom
-    platform_allow: native_posix gd32f450i_eval
+    platform_allow: >
+      gd32f450i_eval
+      native_posix
+      native_posix_64
     integration_platforms:
-      - native_posix
+      - native_posix_64
+    harness: console
+  sample.drivers.eeprom.shell:
+    platform_allow: >
+      gd32f450i_eval
+      native_posix
+      native_posix_64
+    extra_args: OVERLAY_CONFIG=overlay-shell.conf
+    harness: keyboard
   sample.drivers.eeprom.shield.x_nucleo_eeprma2:
-    tags: eeprom shield
+    tags: shield
```

Comment:
Please leave the `tags: eeprom shield` line as-is or the `tags` will be overwritten.